test/boost/json_cql_query_test.cc

```diff
@@ -495,6 +496,83 @@ SEASTAR_TEST_CASE(test_insert_json_implicit_casts) {
                 "pk": 10, "v": [15]
             }'
         )").get(), marshal_exception);
+
+        // Casting to varchar column:
+        cquery_nofail(e, R"(
+            INSERT INTO implicit_casts JSON '{
+                "pk": 1, "v2": 15
+            }'
+        )");
+        cquery_nofail(e, R"(
+            INSERT INTO implicit_casts JSON '{
+                "pk": 2, "v2": 4.81
+            }'
+        )");
+        cquery_nofail(e, R"(
+            INSERT INTO implicit_casts JSON '{
+                "pk": 3, "v2": 834658736487.7474
+            }'
+        )");
+        cquery_nofail(e, R"(
```

Comment:
I think shouldn't be just "nofail", it should check the output for being correct...

Reply:
I don't understand: `INSERT JSON` does not have any output. The inserted row is checked below in `require_rows`.

Reply:
What I really meant to say was that we should probably read after the write, to check what was written... But it's not critical - it will only make the test better. It's already better than nothing.

Reply:
I still don't understand: "we should probably read after the write, to check what was written" - I do perform that read in `require_rows(e, "SELECT pk, v2 FROM implicit_casts", ...`.

Reply:
Ok, I missed the read, sorry.